core/client/client.go

```diff
@@ -74,6 +69,19 @@ var (
 	Vlog = log.New(ioutil.Discard, "", 0)
 )
 
+// Verifier is used to verify specific outputs from Key Transparency.
+type Verifier interface {
+	// Index computes the index from a VRF proof.
+	Index(vrfProof []byte, domainID, appID, userID string) ([]byte, error)
+	// VerifyGetEntryResponse verifies everything about a GetEntryResponse.
+	VerifyGetEntryResponse(ctx context.Context, domainID, appID, userID string, trusted types.LogRootV1, in *pb.GetEntryResponse) (*types.MapRootV1, *types.LogRootV1, error)
+	// VerifyEpoch verifies that epoch is correctly signed and included in the append only log.
+	// VerifyEpoch also verifies that epoch.LogRoot is consistent with the last trusted SignedLogRoot.
```

Comment:
Did the *pb.Epoch parameter used to be called 'epoch' or something?  Maybe capitalize in these comments to make it clear you're talking about the type rather than the variable name - 'VerifyEpoch verifies that the Epoch is correctly...', 'VerifyEpoch also verifies that Epoch.LogRoot is ....'

Reply:
I'll just rename `in` to `epoch` :-)